pygrocy/grocy.py

```diff
@@ -324,6 +326,42 @@ def id(self) -> str:
     def name(self) -> str:
         return self._name
 
+class MealPlanItem(object):
+    def __init__(self, response: MealPlanResponse):
+        self._id = response.id
+        self._day = response.day
+        self._recipe_id = response.recipe_id
+        self._recipe_servings = response.recipe_servings
+        self._note = response.note
+
+    @property
+    def id(self) -> str:
+        return self._id
+
+    @property
+    def day(self) -> datetime:
+        return self._day
+
+    @property
+    def recipe_id(self) -> int:
+        return self._recipe_id
+
+    @property
+    def recipe_servings(self) -> int:
+        return self._recipe_servings
+
+    @property
+    def note(self) -> str:
+        return self._note
+
+    @property
+    def recipe(self) -> RecipeDetailsResponse:
```

Comment:
I'd like to keep "response classes" out of the publicly available API. Could you add an extra class `RecipeDetails` similar to `MealPlanItem` and `MealPlanResponse`?